string model_name
---
geometry_msgs/PoseStamped pose